### PR TITLE
Add athena adapter preview

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adds Amazon Athena adapter configuration page for dbt Cloud (in Preview)
Updates list of available dbt Cloud connections
Adds a release note for the Preview

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

**Comment by amychen1776** [comment]
@matthewshaver I think you're missing something on the sidebar because I can't see Athena on it 
image

File: website/docs/docs/dbt-versions/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 [^*] The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.
 
+## Auguus 2024
```
Comment (amychen1776) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
## August 2024
```

**Review by amychen1776 — APPROVED** [review]
This looks good to me outside of the comments I made

**Review by jenniferjsmmiller — APPROVED** [review]
+1 to Amy's comments, but this looks good to me otherwise!

File: website/docs/docs/cloud/connect-data-platform/about-connections.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ pagination_prev: null
 ---
 dbt Cloud can connect with a variety of data platform providers including: 
 - [AlloyDB](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) 
+- [Amazon Athena (Preview)](/docs/cloud/connect-data-platform/connect-amazon-athena)
```
Comment (matthewshaver) [inline website/docs/docs/cloud/connect-data-platform/about-connections.md]:
```suggestion
- [Amazon Athena (Beta)](/docs/cloud/connect-data-platform/connect-amazon-athena)
```

File: website/docs/docs/cloud/connect-data-platform/about-connections.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ pagination_prev: null
 ---
 dbt Cloud can connect with a variety of data platform providers including: 
 - [AlloyDB](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) 
+- [Amazon Athena (Beta)](/docs/cloud/connect-data-platform/connect-amazon-athena)
```
Comment (mirnawong1) [inline website/docs/docs/cloud/connect-data-platform/about-connections.md:11]: qq - is it preview or beta?

File: website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Connect Amazon Athena"
+id: connect-amazon-athena
+description: "Configure Amazon Athena connection."
```
Comment (mirnawong1) [inline website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md]: sugg adding a little more detail/color
```suggestion
description: "Configure the Amazon Athena data platform connection in dbt Cloud."
```

File: website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+# Connect Amazon Athena <Lifecycle status="beta" />
+
+:::note beta
+
+This is a beta feature with limited availability. A public preview will follow shortly, for wider early access. For more information, check out our [product lifecycle](/docs/dbt-versions/product-lifecycles#dbt-cloud) page. 
+
+:::
+
+Your environment(s) must be on ["Keep on latest version"](/docs/dbt-versions/versionless-cloud) to use the Amazon Athena connection.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md:16]:
```suggestion
Your environment(s) must be on [Keep on latest version](/docs/dbt-versions/versionless-cloud) to use the Amazon Athena connection.
```

File: website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+Connect dbt Cloud to Amazon's Athena interactive query service to build your dbt Project. The following are the required and optional fields for configuring the Athena connection:
```
Comment (mirnawong1) [inline website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md]:
```suggestion
Connect dbt Cloud to Amazon's Athena interactive query service to build your dbt project. The following are the required and optional fields for configuring the Athena connection:
```

Comment (mirnawong1) [inline website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md]: Where should users add this field? maybe a screenshot would be helpful?

**Comment by mirnawong1** [comment]
hey Matt, looks great and added some comments. there's no release note for this - should there be?

**Review by mirnawong1 — APPROVED** [review]
lgtm!

File: website/docs/docs/cloud/connect-data-platform/about-connections.md (hunk 3 above)
Comment (matthewshaver) [reply website/docs/docs/cloud/connect-data-platform/about-connections.md:11]: This is the beta

File: website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md (hunk 5 above)
Comment (matthewshaver) [reply website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md:16]: We've been using the quotes elsewhere, so I'm gonna leave it to remain consistent

**Pull request merged**